docfx_yaml/markdown_utils.py

```diff
@@ -254,7 +259,14 @@ def move_markdown_pages(app: sphinx.application, outdir: Path) -> None:
         'readme.md': 'index.md',
     }
 
-    markdown_dir = Path(app.builder.outdir).parent / "markdown"
+    base_markdown_dir = Path(app.builder.outdir).parent / "markdown"
+
+    markdown_dir = (
+        base_markdown_dir / '/'.join(cwd)
```

Comment:
```suggestion
        base_markdown_dir.joinpath(*cwd)
```

(Using `'/'` and `pathlib` at the same time is a red flag.)